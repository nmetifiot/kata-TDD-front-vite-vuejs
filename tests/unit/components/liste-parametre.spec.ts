import {mount} from "@vue/test-utils";
import {describe, expect, it} from "vitest";
import ListeParametre from "../../../src/components/liste-parametre.vue";

const generateWrapper = () => mount(ListeParametre);

describe('ListeParametre', () => {
    describe('render', () => {
        it('should render correctly', () => {
            const wrapper = generateWrapper();

                expect(wrapper.html()).toMatchSnapshot()
        });
        it('should render a list', () => {
            const wrapper = generateWrapper();

            const list = wrapper.find('[data-testid=liste-parametre__list-container]');

            expect(list.exists()).toBe(true);
        })
    })
})