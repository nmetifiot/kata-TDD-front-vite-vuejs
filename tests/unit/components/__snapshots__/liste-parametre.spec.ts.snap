// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ListeParametre > render > should render liste with data 1`] = `"<div data-testid=\\"liste-parametre__list-container\\"></div>"`;
